utils/README.md

```diff
@@ -27,22 +34,22 @@ Here is an example of the initial configuration file which should be provided to
         {
             "symbol": "XXX",
             "decimals": 18,
-            "supply": "1829101"
+            "supply": 1829101

```

Comment:
different format than Deployment.js - intentional?

Reply:
Nope :(

Reply:
It wasn't intentional, but I've tried to fix it now and realized that JSON doesn't support this :(

Reply:
The reason why I've used quotes for all amounts here is that technically they may exceed `Number.MAX_SAFE_INTEGER`.

Although with the introduction of the `decimals` field, the chances of that are slim, I figured that it would still be generally more robust to keep it that way.

Also, if I remember correctly, the function or functions which handle these amounts, expect a string and not a number, so if we're still using any of these functions then you may want to verify that our code doesn't break during runtime.

Reply:
re: `Number.MAX_SAFE_INTEGER` - `BigNumber` will take care of this and in the worst case - we'd adapt. Working with numbers as strings by default is buggy (see the previous comment + it also opens a room of error when mixing hex or even non-ascii). I've refactored the script to expect numbers where numbers are due in any case.

Reply:
In contrast with `BigNumber('111111111...')`, which is guaranteed to work correctly, I doubt that `BigNumber(11111111...)` is safe. The constructor function will have already received the inaccurate numeric value when called.

Reply:
It'll throw an exception and we would be able to adapt. Currently, our config files don't have these values though (in other words, let's handle the exception explicitly rather than the other way around)

Reply:
AFAIK, using very large numbers in native format doesn't cause any exception, but rather an inaccurate interpretation of that number during runtime.

Reply:
You're thinking about the older library we were using.

Reply:
It's general to JS AFAIK (not related to a specific package). A number larger than `Number.MAX_SAFE_INTEGER` is not safe in terms of accuracy.

Reply:
AFAIK, it is handled by "Big Number" libraries exactly just for that